test_util/cluster_api.py

```diff
@@ -187,7 +177,7 @@ def _wait_for_metronome(self):
 
     def wait_for_dcos(self):
         self._wait_for_adminrouter_up()
-        if self.auth_enabled and self.web_auth_default_user:
+        if self.web_auth_default_user:
```

Comment:
`if self.web_auth_default_user is not None`:

(Since we use the default of None).